app/containers/Dashboard/txnsToList.js

```diff
@@ -6,16 +6,15 @@ import BigNumber from 'bignumber.js';
 import A from '../../components/A';
 import WithLoading from '../../components/WithLoading';
 import { conf } from '../../app.config';
-import { formatEth, formatNtz, formatAbp } from '../../utils/amountFormatter';
+import { formatEth, formatNtz } from '../../utils/amountFormatter';
 
 import { Icon, TypeIcon, Error, ErrorIcon, typeIcons } from './styles';
 import messages from './messages';
 import {
   isSellEvent,
   isETHPayoutEvent,
-  isABPPayoutEvent,
-  isPurchaseStartEvent, isPurchaseEndEvent,
-  isPowerUpEvent,
+  isPurchaseStartEvent,
+  isPurchaseEndEvent,
```

Comment:
Purchase/sell stuff also can be safely removed

Reply:
ok, thanks for the tips^^